src/zones.py

```diff
@@ -0,0 +1,39 @@
+import cv2
+
+class Zone:
+    def __init__(self):
+        self.x = -1
+        self.y = -1
+        self.width = -1
+        self.height = -1
+        self.drawing = False
+        self.rectReady = False
+
+    def setSquare(self, event, x, y, flags, params):
```

Comment:
interesting, does the user draw the square?

Reply:
Yes indeed! They user will ultimately be in charge of what zones they want to monitor.